src/Adapter/Language/LanguagePackInstaller.php

```diff
@@ -50,11 +44,9 @@ final class LanguagePackInstaller implements LanguagePackInstallerInterface
      * LanguagePackInstaller constructor.
      *
      * @param TranslatorInterface $translator
-     * @param Version $version
      */
-    public function __construct(TranslatorInterface $translator, Version $version)
+    public function __construct(TranslatorInterface $translator)
```

Comment:
Same here

Reply:
I don't see the value of removing this and create a breaking change here

Reply:
So did you guys agree on removing or keeping this?

@MeKeyCool : about keeping old unused parameters. Although it looks like a good thing for us, it becomes a possible trouble from people trying to update the code that rely on PS so we have to balance things. If you had a hard time in the past upgrading an app from Symfony 1 to Symfony 2 or Laravel 3 to Laravel 4 ... well today you can make it easier for people to upgrade from PS 1.7 to 8 by keeping this parameter here 😄  welcome on SemVer side!